updatest to read and remove coupons from an array of strings instead … (#14)
dove-technology/commercetools-campaigns-connector-demo: README.md

The main area of customisation is the coupon codes functionality. commercetools doesn't allow changing the provider of coupon codes so we need to store them in a custom field on the cart and update the UI to use this.

> Coupon codes are stored on the cart using the `dovetech-discounts-coupon-codes` custom field. This is an array of coupon code objects serialised in JSON.

Reviewer: ```suggestion
Coupon codes are stored on the cart using the `dovetech-discounts-coupon-codes` custom field. The type of this field is a set of strings.
```
